k6/scripts/load.js

```diff
@@ -0,0 +1,56 @@
+import http from 'k6/http';
+import { check, group, sleep, fail } from 'k6';
+
+export let options = {
+    stages: [
+        { duration: '1m', target: 3 },
+        { duration: '2m', target: 7 },
+        { duration: '3m', target: 13 },
```

Comment:
```suggestion
        { duration: '10s', target: 3 },
        { duration: '1m', target: 3 },
        { duration: '10s', target: 7 },        
        { duration: '2m', target: 7 },
        { duration: '10s', target: 13 },        
        { duration: '3m', target: 13 },
```

k6 는 duration 기간 동안 target 수치 만큼 점진적으로 부하를 늘려나갑니다.
13 값으로 일정기간 부하를 주고 싶다면 위와 같이 구성하시면 되어요~